Gems/Atom/Feature/Common/Assets/Passes/MainPipeline.pass

```diff
@@ -41,7 +41,10 @@
                 },
                 {
                     "Name": "RayTracingAccelerationStructurePass",
-                    "TemplateName": "RayTracingAccelerationStructurePassTemplate"
+                    "TemplateName": "RayTracingAccelerationStructurePassTemplate",
+                    "ExecuteAfter": [
+                        "SkinningPass"
```

Comment:
This does not guarantee execution order on the gpu. In fact we should remove support for ExecuteAfter/ExecuteBefore completely as it gives a false sense of dependency to the developer. The gpu execution dependency can only be established via framegraph, so in this case this dependency should only work if the output of skinning pass is an input attachment for the RT pass. 
```suggestion
                    "ExecuteAfter": [
                        "SkinningPass"
```